l10n_it_pec_messages/model/mail_thread.py

```diff
@@ -102,11 +111,72 @@ def get_pec_attachments(self, cr, uid, message, context=None):
                     smime = attachment
         return (postacert, daticert, smime)
 
+    def _message_extract_payload_receipt(self, message,
+                                         save_original=False):
+        """Extract body as HTML and attachments from the mail message"""
```

Comment:
La necessità di utilizzare questo metodo nasce dal fatto che i messaggi di consegna vengono gestiti in modo non corretto dal metodo _message_extract_payload di base. Mi spiego: se il messaggio è multipart e presenta una parte di body in testo normale ed una in testo html, il sistema prima setta body = txt e poi sovrascrive con body = html.
Per i messaggi di consegna il corpo del testo è solo testo ma nel multi-part è presente anche il riferimento al messaggio principale che era stato spedito in formato html. Il sistema quindi prende quello come messaggio corretto e sovrascrive le info di consegna con il messaggio originale che era stato spedito. Da qui la necessità di gestire in modo diverso il parsing del body del messaggio di consegna